packages/core/src/routes/subject-token.openapi.json

```diff
@@ -1,13 +1,13 @@
 {
   "tags": [
     {
-      "name": "Security",
-      "description": "Security related endpoints."
+      "name": "Subject tokens",
```

Comment:
just double check if "additionalTags" in the "packages/core/src/routes/swagger/index.ts" has been updated accordingly

Reply:
updated